src/lwc/gsAdminSetup/gsAdminSetup.js

```diff
@@ -0,0 +1,25 @@
+import { LightningElement, track } from 'lwc';
+
+import gsAdminSetupTitle from '@salesforce/label/c.gsAdminSetupTitle'
+
+import exploreNonProfit from './exploreNonprofit';
+import makeYou from './makeYou';
```

Comment:
`makeYou` seems like a fragment of a title. How about something like `makeItYours` for everything that just has "makeyou". This probably applies to the label descriptions as well where "Make You" is used. By itself that doesn't mean anything. If I'm reading the PR correct it's really "Make It Yours", not "Make You" or "Make Your".